README.md

```diff
@@ -1,288 +1,178 @@
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Test</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://rspec.info/">Ruby</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
+### Live Demo
+Here is our [Project Demonstration](#)
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+front end react path [Project path](https://github.com/frnega/hello-react-front-end)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Running the app
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
+rails db:create:all
+rails db:migrate
+rails db:seed ### to populate the database with some sample data.
+rails s ## to start the server.
+http://127.0.0.1:3000/api/messages/random_greeting
+### Key Features <a name="key-features"></a>
 
-- [Live Demo Link](https://google.com)
+- **random generation of message**
+- **use webpack**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
 
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  cd PATH
+  git clone https://github.com/frnega/hello-rails-back-end
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
+  cd Recipe-app
   gem install
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
   rails server
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
+rspec
 ```sh
-  bin/rails test test/models/article_test.rb
+  
 ```
---->
-
-### Deployment
 
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **FREHIYWOT Nega**
+- GitHub: [FREHIYWOT Nega](https://github.com/frnega)
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if you add your LinkedIn and twitter account handle in this section to make yourself easily reachable by the viewers or recruiters :+1: